Vagrantfile

```
@@ -31,19 +31,11 @@ EOF
         && mkdir /busybox \
         && curl -fsSL $(get_busybox) | tar xfJC - /busybox
 
-    # TODO: remove this after criu 3.14 is released
-    rpm -e --nodeps criu || true
-    CRIU_VERSION=v3.13
-    mkdir -p /usr/src/criu \
-        && curl -fsSL https://github.com/checkpoint-restore/criu/archive/${CRIU_VERSION}.tar.gz | tar -C /usr/src/criu/ -xz --strip-components=1 \
-        && cd /usr/src/criu \
-        && echo 1 > .gitid \
-        && curl -sSL https://github.com/checkpoint-restore/criu/commit/4c27b3db4f4325a311d8bfa9a50ea3efb4d6e377.patch | patch -p1 \
-        && curl -sSL https://github.com/checkpoint-restore/criu/commit/aac41164b2cd7f0d2047f207b32844524682e43f.patch | patch -p1 \
-        && curl -sSL https://github.com/checkpoint-restore/criu/commit/6f19249b2565f3f7c0a1f8f65b4ae180e8f7f34b.patch | patch -p1 \
-        && curl -sSL https://github.com/checkpoint-restore/criu/commit/378337a496ca759848180bc5411e4446298c5e4e.patch | patch -p1 \
-        && make install-criu \
-        && cd - \
-        && rm -rf /usr/src/criu
+    # Apr 22, 2020
+    ( git clone https://github.com/checkpoint-restore/criu.git /usr/src/criu \
+     && cd /usr/src/criu \
+     && git checkout 27fa80634adfafd0f49d756e621943c5cea33532 \
```

We need to use the master branch, not the criu-dev branch. IOW
```suggestion
     && git checkout 5c5e7695a51318b17e3d982df8231ac83971641c \
```

---

thanks

---

I still can't repro this locally (on my F32 laptop) with any criu which can be compiled.

Will try on vagrant box tomorrow